tests/eval_tests/utils.py

```diff
@@ -143,6 +184,53 @@ def add_plot(fig, ax, df, column_name, title, save_filename=True):
     if save_filename:
         save_plot(fig, title)
 
+def add_per_tool_plot(df, column_name, title):
```

Comment:
This is another case where the function is very similar to `add_plot()` except its tracking "expected_tool_call" instead of "model". Do you think you could update this function to be used for both cases?